jenkins/fossa_debug.ps1

```diff
@@ -3,6 +3,19 @@
 [CmdletBinding()]
 Param()
 
+# DEPRECATION NOTICE
+Write-Output "******************************************************************"
+Write-Output "  NOTICE : THIS SCRIPT HAS BEEN PULLED FROM A DEPRECATED SOURCE"
+Write-Output "           PLEASE SEE :"
+Write-Output "  https://github.com/mdsol/fossa_ci_scripts/blob/main/jenkins/SETUP.md"
+Write-Output "******************************************************************"
```

Comment:
Should we use `Write-Error` commandlet for the powershell scripts?

Reply:
I generally only use Write-Output so that its destination is based on the pipeline that its attached to... and it keeps it consistent with the rest of the script.